modules/nodes-adding-bare-metal-host-to-cluster-using-cli.adoc

```diff
@@ -0,0 +1,73 @@
+// Module included in the following assemblies:
+//
+// nodes/nodes/nodes-managing-bare-metal-nodes.adoc
+
+[id="nodes-adding-bare-metal-host-to-cluster-using-cli_{context}"]
+= Adding a bare metal host to the cluster and provisioning a bare metal node using the CLI
+
+You can add bare metal hosts to the cluster and then provision them as bare metal nodes using the CLI.
+
+.Prerequisites
+
+* Install a cluster on bare metal, and provision a new bare metal host for use in the cluster.
```

Comment:
"Provision a new bare metal host for use in the cluster" is what this section is describing how to do, so it doesn't seem to be a prerequisite.

Reply:
Maybe the sentence is unclear - the host must be provisioned with RHOCS, BMC address, etc before we add it to the cluster as a node right?